libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/DefaultDashboardAttributeFilter.tsx

```diff
@@ -170,12 +170,8 @@ export const DefaultDashboardAttributeFilter = (
         };
     }, [isConfigurationOpen, filterRef, filterByText, displayValuesAsText]);
 
-    if (!attributes) {
-        return null;
-    }
-
     return (
-        <AttributeFilterParentFilteringProvider filter={filter} attributes={attributes}>
+        <AttributeFilterParentFilteringProvider filter={filter} attributes={attributes ?? []}>
```

Comment:
can you memo attributes in case its undefined ? New array is new dependency every render.

Reply:
Hope it will no problem here 
https://github.com/gooddata/gooddata-ui-sdk/pull/3525/files#diff-deeb9561777e06c60da6f9a853f6168b111aec8b1d729c2e07bb1b09d65cdd72R128